inc/customizer/class-storefront-customizer.php

```diff
@@ -786,11 +786,12 @@ public function get_css() {
 				color: ' . $storefront_theme_mods['footer_text_color'] . ';
 			}
 
-			.site-footer a:not(.button) {
+			.site-footer a:not(.button),
```

Comment:
It should be safe to remove this line now, the next one has more specificity and we don't want to change the color of button components anyway (elements with the `.components-button` class).
```suggestion
```

Reply:
Thanks, @senadir, for adjusting this in https://github.com/woocommerce/storefront/pull/1261/commits/001c3a4324966e32437eb0c7d499f9bf874f3cd5 😉